### DISCUSSION TITLE
Add instruction for running a specific test from a specific file

### DISCUSSION THREAD

**Discussion by bytrangle** [decaporg/decap-cms#6298] [Ideas]
Sorry if this hasn't been mentioned in the Contributing guideline.

Sometimes, a few code changes one made leads to some unit tests, or test suits to fail. AFAIK, you can run a specific file with this command:

```
yarn jest FILENAME
````
For example: `yarn jest gitlab.spec.js` to run all the tests inside the `gitlab.spec.js` file.

You can also run a specific test inside a file by adding the flag `--testNamePattern`.

```
yarn jest --testNamePattern "stores" gitlab.spec.js
```

The command above will run the test that contains the word "stores` in the `gitlab.spec.js`.

I understand that this is peculiar to Jest and not in the domain of Netlify CMS, so I wonder if we should add the tip above to the Contributing guideline? Until a few days ago, I didn't know any better and troubleshoot one failed file, then ran `yarn test` and waited for all of them to finish 😨.

**Comment by erezrokah** [comment]
@bytrangle, this is a great idea! Please do add it 🚀
